nemo_skills/inference/model/base.py

```diff
@@ -130,6 +148,11 @@ def _maybe_apply_stop_phrase_removal(
         if remove_stop_phrases:
             result["generation"] = trim_after_stop_phrases(result["generation"], stop_phrases)
 
+    @abc.abstractmethod
```

Comment:
this is not really abstract as by default it just disables the functionality and isn't required
```suggestion
```